pkg/cmd/server/api/types.go

```diff
@@ -968,6 +968,11 @@ type KubernetesMasterConfig struct {
 	// the server will not start. These values may override other settings in KubernetesMasterConfig which may cause invalid
 	// configurations.
 	ControllerArguments ExtendedArguments
+
+	// ReconcileMasterEndpoints indicates that OpenShift should update the list of endpoints for the
+	// "kubernetes" Service, removing any entry from the list whose corresponding master/apiserver is
+	// unhealthy or down.
+	ReconcileMasterEndpoints bool
```

Comment:
> (although how do we handle the fact that we're adding a new bool that needs to default to true in the config?)

You'll need to make it a `*bool` and then handle the cases in the code that uses it.

@smarterclayton we have more time here than upstream, you want to just leave it unconditionally on until we tag?